jetty-ee11/jetty-ee11-websocket/jetty-ee11-websocket-jakarta-tests/src/test/java/org/eclipse/jetty/ee11/websocket/jakarta/tests/JakartaClientClassLoaderTest.java

```diff
@@ -183,7 +183,7 @@ public void websocketProvidedByServer() throws Exception
         assertThat(response.getStatus(), is(HttpStatus.OK_200));
 
         // The ContextClassLoader in the WebSocketClients onOpen was the WebAppClassloader.
-        assertThat(response.getContentAsString(), containsString("ContextClassLoader: oeje.WebAppClassLoader"));
+        assertThat(response.getContentAsString(), containsString("ContextClassLoader: oejew.WebAppClassLoader"));
```

Comment:
Couldn't these kinds of assertions be changed to use TypeUtil to avoid this kind of change?

```suggestion
        assertThat(response.getContentAsString(), containsString("ContextClassLoader: " + TypeUtil.toShortName(WebAppClassLoader.class));
```